
time="2025-07-30T19:08:26-03:00" level=warning msg="/home/rinha-user/rinha-de-backend-2025/participantes/Maxel-Uds-quarkus/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion"
 load_balancer Pulling 
 api01 Pulling 
 api02 Pulling 
 fcce199ebec0 Already exists 
 4f4fb700ef54 Already exists 
 ec661f466405 Already exists 
 b76405cb56cd Already exists 
 9822410229eb Pulling fs layer 
 52a298e39d05 Pulling fs layer 
 da3cc40e82e4 Pulling fs layer 
 fcce199ebec0 Already exists 
 4f4fb700ef54 Already exists 
 ec661f466405 Already exists 
 b76405cb56cd Already exists 
 42c077c10790 Already exists 
 9822410229eb Pulling fs layer 
 52a298e39d05 Pulling fs layer 
 da3cc40e82e4 Pulling fs layer 
 992fa0855102 Already exists 
 94ad8e5864b2 Already exists 
 9bfcb21bb22b Already exists 
 ac27b0060bc3 Pulling fs layer 
 4f24567a4129 Pulling fs layer 
 e4fa8a451c60 Pulling fs layer 
 bd43178e6ad6 Pulling fs layer 
 ac27b0060bc3 Waiting 
 4f24567a4129 Waiting 
 e4fa8a451c60 Waiting 
 bd43178e6ad6 Waiting 
 9822410229eb Downloading [==================================================>]      94B/94B
 9822410229eb Download complete 
 9822410229eb Verifying Checksum 
 9822410229eb Download complete 
 9822410229eb Extracting [==================================================>]      94B/94B
 9822410229eb Extracting [==================================================>]      94B/94B
 9822410229eb Extracting [==================================================>]      94B/94B
 9822410229eb Extracting [==================================================>]      94B/94B
 9822410229eb Pull complete 
 9822410229eb Pull complete 
 da3cc40e82e4 Downloading [>                                                  ]  303.1kB/30.01MB
 da3cc40e82e4 Downloading [>                                                  ]  303.1kB/30.01MB
 52a298e39d05 Downloading [==================================================>]     101B/101B
 52a298e39d05 Verifying Checksum 
 52a298e39d05 Download complete 
 52a298e39d05 Extracting [==================================================>]     101B/101B
 52a298e39d05 Verifying Checksum 
 52a298e39d05 Download complete 
 52a298e39d05 Extracting [==================================================>]     101B/101B
 52a298e39d05 Extracting [==================================================>]     101B/101B
 52a298e39d05 Extracting [==================================================>]     101B/101B
 52a298e39d05 Pull complete 
 52a298e39d05 Pull complete 
 da3cc40e82e4 Downloading [=>                                                 ]  909.3kB/30.01MB
 da3cc40e82e4 Downloading [=>                                                 ]  909.3kB/30.01MB
 da3cc40e82e4 Downloading [===>                                               ]  2.138MB/30.01MB
 da3cc40e82e4 Downloading [===>                                               ]  2.138MB/30.01MB
 da3cc40e82e4 Downloading [=======>                                           ]  4.579MB/30.01MB
 da3cc40e82e4 Downloading [=======>                                           ]  4.579MB/30.01MB
 da3cc40e82e4 Downloading [================>                                  ]  10.06MB/30.01MB
 da3cc40e82e4 Downloading [================>                                  ]  10.06MB/30.01MB
 da3cc40e82e4 Downloading [==============================>                    ]  18.01MB/30.01MB
 da3cc40e82e4 Downloading [==============================>                    ]  18.01MB/30.01MB
 da3cc40e82e4 Downloading [==========================================>        ]  25.62MB/30.01MB
 da3cc40e82e4 Downloading [==========================================>        ]  25.62MB/30.01MB
 da3cc40e82e4 Download complete 
 da3cc40e82e4 Verifying Checksum 
 da3cc40e82e4 Download complete 
 da3cc40e82e4 Extracting [>                                                  ]  327.7kB/30.01MB
 da3cc40e82e4 Extracting [>                                                  ]  327.7kB/30.01MB
 da3cc40e82e4 Extracting [============>                                      ]  7.537MB/30.01MB
 da3cc40e82e4 Extracting [============>                                      ]  7.537MB/30.01MB
 da3cc40e82e4 Extracting [======================>                            ]  13.76MB/30.01MB
 da3cc40e82e4 Extracting [======================>                            ]  13.76MB/30.01MB
 ac27b0060bc3 Downloading [==================================================>]  1.002kB/1.002kB
 ac27b0060bc3 Verifying Checksum 
 ac27b0060bc3 Download complete 
 ac27b0060bc3 Extracting [==================================================>]  1.002kB/1.002kB
 ac27b0060bc3 Extracting [==================================================>]  1.002kB/1.002kB
 da3cc40e82e4 Extracting [================================>                  ]  19.33MB/30.01MB
 da3cc40e82e4 Extracting [================================>                  ]  19.33MB/30.01MB
 ac27b0060bc3 Pull complete 
 da3cc40e82e4 Extracting [===========================================>       ]  25.89MB/30.01MB
 da3cc40e82e4 Extracting [===========================================>       ]  25.89MB/30.01MB
 da3cc40e82e4 Extracting [==================================================>]  30.01MB/30.01MB
 da3cc40e82e4 Extracting [==================================================>]  30.01MB/30.01MB
 da3cc40e82e4 Pull complete 
 da3cc40e82e4 Pull complete 
 api01 Pulled 
 api02 Pulled 
 4f24567a4129 Downloading [>                                                  ]  532.5kB/119.9MB
 4f24567a4129 Downloading [>                                                  ]  1.597MB/119.9MB
 4f24567a4129 Downloading [=>                                                 ]  3.736MB/119.9MB
 e4fa8a451c60 Downloading [>                                                  ]  2.738kB/233.2kB
 4f24567a4129 Downloading [===>                                               ]  7.475MB/119.9MB
 e4fa8a451c60 Downloading [==================================================>]  233.2kB/233.2kB
 e4fa8a451c60 Verifying Checksum 
 e4fa8a451c60 Download complete 
 4f24567a4129 Downloading [======>                                            ]   14.4MB/119.9MB
 bd43178e6ad6 Downloading [==================================================>]     440B/440B
 bd43178e6ad6 Verifying Checksum 
 bd43178e6ad6 Download complete 
 4f24567a4129 Downloading [==========>                                        ]  24.01MB/119.9MB
 4f24567a4129 Downloading [==============>                                    ]  35.76MB/119.9MB
 4f24567a4129 Downloading [===================>                               ]  47.53MB/119.9MB
 4f24567a4129 Downloading [========================>                          ]  59.81MB/119.9MB
 4f24567a4129 Downloading [============================>                      ]  68.91MB/119.9MB
 4f24567a4129 Downloading [============================>                      ]  69.45MB/119.9MB
 4f24567a4129 Downloading [=============================>                     ]  70.51MB/119.9MB
 4f24567a4129 Downloading [==============================>                    ]  72.63MB/119.9MB
 4f24567a4129 Downloading [================================>                  ]  77.43MB/119.9MB
 4f24567a4129 Downloading [===================================>               ]  85.95MB/119.9MB
 4f24567a4129 Downloading [========================================>          ]  97.15MB/119.9MB
 4f24567a4129 Downloading [============================================>      ]  107.3MB/119.9MB
 4f24567a4129 Downloading [=================================================> ]  119.6MB/119.9MB
 4f24567a4129 Verifying Checksum 
 4f24567a4129 Download complete 
 4f24567a4129 Extracting [>                                                  ]  557.1kB/119.9MB
 4f24567a4129 Extracting [==>                                                ]  5.014MB/119.9MB
 4f24567a4129 Extracting [=====>                                             ]  12.81MB/119.9MB
 4f24567a4129 Extracting [========>                                          ]  20.05MB/119.9MB
 4f24567a4129 Extracting [========>                                          ]  21.17MB/119.9MB
 4f24567a4129 Extracting [=========>                                         ]  22.28MB/119.9MB
 4f24567a4129 Extracting [============>                                      ]  29.52MB/119.9MB
 4f24567a4129 Extracting [==============>                                    ]  33.98MB/119.9MB
 4f24567a4129 Extracting [================>                                  ]  40.67MB/119.9MB
 4f24567a4129 Extracting [====================>                              ]  48.46MB/119.9MB
 4f24567a4129 Extracting [=====================>                             ]  51.81MB/119.9MB
 4f24567a4129 Extracting [=======================>                           ]  56.26MB/119.9MB
 4f24567a4129 Extracting [==========================>                        ]   63.5MB/119.9MB
 4f24567a4129 Extracting [=============================>                     ]  70.19MB/119.9MB
 4f24567a4129 Extracting [================================>                  ]  76.87MB/119.9MB
 4f24567a4129 Extracting [==================================>                ]  83.56MB/119.9MB
 4f24567a4129 Extracting [=====================================>             ]  89.13MB/119.9MB
 4f24567a4129 Extracting [======================================>            ]  91.36MB/119.9MB
 4f24567a4129 Extracting [=======================================>           ]  93.59MB/119.9MB
 4f24567a4129 Extracting [========================================>          ]  96.37MB/119.9MB
 4f24567a4129 Extracting [========================================>          ]  97.48MB/119.9MB
 4f24567a4129 Extracting [=========================================>         ]  99.71MB/119.9MB
 4f24567a4129 Extracting [=========================================>         ]  100.3MB/119.9MB
 4f24567a4129 Extracting [==========================================>        ]  101.4MB/119.9MB
 4f24567a4129 Extracting [==========================================>        ]  103.1MB/119.9MB
 4f24567a4129 Extracting [===========================================>       ]  104.7MB/119.9MB
 4f24567a4129 Extracting [================================================>  ]  115.9MB/119.9MB
 4f24567a4129 Extracting [==================================================>]  119.9MB/119.9MB
 4f24567a4129 Pull complete 
 e4fa8a451c60 Extracting [=======>                                           ]  32.77kB/233.2kB
 e4fa8a451c60 Extracting [==================================================>]  233.2kB/233.2kB
 e4fa8a451c60 Extracting [==================================================>]  233.2kB/233.2kB
 e4fa8a451c60 Pull complete 
 bd43178e6ad6 Extracting [==================================================>]     440B/440B
 bd43178e6ad6 Extracting [==================================================>]     440B/440B
 bd43178e6ad6 Pull complete 
 load_balancer Pulled 
 Network maxel-uds-quarkus_payment-router  Creating
 Network maxel-uds-quarkus_payment-router  Created
 Container postgres  Creating
 Container postgres  Created
 Container api2  Creating
 Container api1  Creating
 Container api2  Created
 Container api1  Created
 Container load_balancer  Creating
 Container load_balancer  Created
Attaching to api1, api2, load_balancer, postgres
postgres       | The files belonging to this database system will be owned by user "postgres".
postgres       | This user must also own the server process.
postgres       | 
postgres       | The database cluster will be initialized with locale "en_US.utf8".
postgres       | The default database encoding has accordingly been set to "UTF8".
postgres       | The default text search configuration will be set to "english".
postgres       | 
postgres       | Data page checksums are disabled.
postgres       | 
postgres       | fixing permissions on existing directory /var/lib/postgresql/data ... ok
postgres       | creating subdirectories ... ok
postgres       | selecting dynamic shared memory implementation ... posix
postgres       | selecting default max_connections ... 100
postgres       | selecting default shared_buffers ... 128MB
postgres       | selecting default time zone ... Etc/UTC
postgres       | creating configuration files ... ok
postgres       | running bootstrap script ... ok
postgres       | performing post-bootstrap initialization ... ok
postgres       | initdb: warning: enabling "trust" authentication for local connections
postgres       | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
postgres       | syncing data to disk ... ok
postgres       | 
postgres       | 
postgres       | Success. You can now start the database server using:
postgres       | 
postgres       |     pg_ctl -D /var/lib/postgresql/data -l logfile start
postgres       | 
postgres       | waiting for server to start....2025-07-30 22:08:45.333 UTC [48] LOG:  starting PostgreSQL 16.9 (Debian 16.9-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres       | 2025-07-30 22:08:45.339 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres       | 2025-07-30 22:08:45.408 UTC [51] LOG:  database system was shut down at 2025-07-30 22:08:44 UTC
postgres       | 2025-07-30 22:08:45.419 UTC [48] LOG:  database system is ready to accept connections
postgres       |  done
postgres       | server started
postgres       | CREATE DATABASE
postgres       | 
postgres       | 
postgres       | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
postgres       | DROP TABLE
postgres       | psql:/docker-entrypoint-initdb.d/init.sql:1: NOTICE:  table "payments" does not exist, skipping
postgres       | CREATE TABLE
postgres       | 
postgres       | 
postgres       | waiting for server to shut down...2025-07-30 22:08:46.220 UTC [48] LOG:  received fast shutdown request
postgres       | .2025-07-30 22:08:46.227 UTC [48] LOG:  aborting any active transactions
postgres       | 2025-07-30 22:08:46.229 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
postgres       | 2025-07-30 22:08:46.302 UTC [49] LOG:  shutting down
postgres       | 2025-07-30 22:08:46.307 UTC [49] LOG:  checkpoint starting: shutdown immediate
postgres       | 2025-07-30 22:08:46.416 UTC [49] LOG:  checkpoint complete: wrote 922 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.026 s, sync=0.055 s, total=0.114 s; sync files=303, longest=0.034 s, average=0.001 s; distance=4263 kB, estimate=4263 kB; lsn=0/19140B0, redo lsn=0/19140B0
postgres       | 2025-07-30 22:08:46.422 UTC [48] LOG:  database system is shut down
postgres       |  done
postgres       | server stopped
postgres       | 
postgres       | PostgreSQL init process complete; ready for start up.
postgres       | 
postgres       | 2025-07-30 22:08:46.564 UTC [1] LOG:  starting PostgreSQL 16.9 (Debian 16.9-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres       | 2025-07-30 22:08:46.564 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres       | 2025-07-30 22:08:46.602 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres       | 2025-07-30 22:08:46.614 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres       | 2025-07-30 22:08:46.630 UTC [66] LOG:  database system was shut down at 2025-07-30 22:08:46 UTC
postgres       | 2025-07-30 22:08:46.641 UTC [1] LOG:  database system is ready to accept connections
load_balancer  | 2025/07/30 22:08:57 [error] 7#7: *4 open() "/usr/local/openresty/nginx/html/purge-payments" failed (2: No such file or directory), client: 172.20.0.1, server: , request: "POST /purge-payments HTTP/1.1", host: "localhost:9999"
[Kload_balancer exited with code 0
api2           | __  ____  __  _____   ___  __ ____  ______ 
api2           |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api2           |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api2           | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api2           | 2025-07-30 22:10:08,948 ERROR [org.jbo.thr.errors] (executor-thread-8) Thread Thread[#43,executor-thread-8,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,951 ERROR [org.jbo.thr.errors] (executor-thread-5) Thread Thread[#40,executor-thread-5,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,951 ERROR [org.jbo.thr.errors] (executor-thread-3) Thread Thread[#38,executor-thread-3,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,948 ERROR [org.jbo.thr.errors] (executor-thread-13) Thread Thread[#48,executor-thread-13,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,951 ERROR [org.jbo.thr.errors] (executor-thread-4) Thread Thread[#39,executor-thread-4,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,948 ERROR [org.jbo.thr.errors] (executor-thread-15) Thread Thread[#50,executor-thread-15,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,951 ERROR [org.jbo.thr.errors] (executor-thread-6) Thread Thread[#41,executor-thread-6,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,952 ERROR [org.jbo.thr.errors] (executor-thread-1) Thread Thread[#36,executor-thread-1,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,952 ERROR [org.jbo.thr.errors] (executor-thread-16) Thread Thread[#51,executor-thread-16,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,948 ERROR [org.jbo.thr.errors] (executor-thread-11) Thread Thread[#46,executor-thread-11,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,951 ERROR [org.jbo.thr.errors] (executor-thread-7) Thread Thread[#42,executor-thread-7,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,951 ERROR [org.jbo.thr.errors] (executor-thread-12) Thread Thread[#47,executor-thread-12,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,952 ERROR [org.jbo.thr.errors] (executor-thread-2) Thread Thread[#37,executor-thread-2,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,952 ERROR [org.jbo.thr.errors] (executor-thread-14) Thread Thread[#49,executor-thread-14,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,952 ERROR [org.jbo.thr.errors] (executor-thread-9) Thread Thread[#44,executor-thread-9,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api2           | 2025-07-30 22:10:08,951 ERROR [org.jbo.thr.errors] (executor-thread-10) Thread Thread[#45,executor-thread-10,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api2           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api2           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api2           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api2           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api2           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api2           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api2           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api2           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api2           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api2           | Caused by: java.lang.InterruptedException
api2           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api2           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api2           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api2           | 	... 13 more
api2           | 
api1           | __  ____  __  _____   ___  __ ____  ______ 
api1           |  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
api1           |  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
api1           | --\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
api1           | 2025-07-30 22:10:08,958 ERROR [org.jbo.thr.errors] (executor-thread-8) Thread Thread[#43,executor-thread-8,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,958 ERROR [org.jbo.thr.errors] (executor-thread-6) Thread Thread[#41,executor-thread-6,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,958 ERROR [org.jbo.thr.errors] (executor-thread-1) Thread Thread[#36,executor-thread-1,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-11) Thread Thread[#46,executor-thread-11,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-5) Thread Thread[#40,executor-thread-5,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-10) Thread Thread[#45,executor-thread-10,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-12) Thread Thread[#47,executor-thread-12,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-9) Thread Thread[#44,executor-thread-9,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,958 ERROR [org.jbo.thr.errors] (executor-thread-16) Thread Thread[#51,executor-thread-16,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,958 ERROR [org.jbo.thr.errors] (executor-thread-4) Thread Thread[#39,executor-thread-4,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-2) Thread Thread[#37,executor-thread-2,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,958 ERROR [org.jbo.thr.errors] (executor-thread-13) Thread Thread[#48,executor-thread-13,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,966 ERROR [org.jbo.thr.errors] (executor-thread-14) Thread Thread[#49,executor-thread-14,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,966 ERROR [org.jbo.thr.errors] (executor-thread-15) Thread Thread[#50,executor-thread-15,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-3) Thread Thread[#38,executor-thread-3,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
api1           | 2025-07-30 22:10:08,965 ERROR [org.jbo.thr.errors] (executor-thread-7) Thread Thread[#42,executor-thread-7,5,main] threw an uncaught exception [Error Occurred After Shutdown]: java.lang.RuntimeException: java.lang.InterruptedException
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:43)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.lambda$onStart$0(PaymentProcessorWorker.java:36)
api1           | 	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
api1           | 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:604)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
api1           | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
api1           | 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
api1           | 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
api1           | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
api1           | 	at java.base@21.0.8/java.lang.Thread.runWith(Thread.java:1596)
api1           | 	at java.base@21.0.8/java.lang.Thread.run(Thread.java:1583)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
api1           | 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
api1           | Caused by: java.lang.InterruptedException
api1           | 	at java.base@21.0.8/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1727)
api1           | 	at java.base@21.0.8/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
api1           | 	at org.payment.router.worker.PaymentProcessorWorker.getNextPayment(PaymentProcessorWorker.java:41)
api1           | 	... 13 more
api1           | 
[Kapi2 exited with code 143
[Kapi1 exited with code 143
postgres       | 2025-07-30 22:10:09.970 UTC [1] LOG:  received fast shutdown request
postgres       | 2025-07-30 22:10:09.978 UTC [1] LOG:  aborting any active transactions
postgres       | 2025-07-30 22:10:09.980 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
postgres       | 2025-07-30 22:10:09.980 UTC [64] LOG:  shutting down
postgres       | 2025-07-30 22:10:09.992 UTC [64] LOG:  checkpoint starting: shutdown immediate
postgres       | 2025-07-30 22:10:10.096 UTC [64] LOG:  checkpoint complete: wrote 404 buffers (2.5%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.033 s, sync=0.049 s, total=0.117 s; sync files=24, longest=0.032 s, average=0.003 s; distance=4902 kB, estimate=4902 kB; lsn=0/1DDDAD0, redo lsn=0/1DDDAD0
postgres       | 2025-07-30 22:10:10.102 UTC [1] LOG:  database system is shut down
[Kpostgres exited with code 0
log truncated at line 1000
